presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -257,6 +261,8 @@
 import static java.util.stream.Collectors.toSet;
 import static org.apache.hadoop.hive.metastore.TableType.EXTERNAL_TABLE;
 import static org.apache.hadoop.hive.metastore.TableType.MANAGED_TABLE;
+import static org.apache.iceberg.BaseMetastoreTableOperations.ICEBERG_TABLE_TYPE_VALUE;
```

Comment:
Since these are the only usages of the Iceberg library, it would be better to manually add the constants rather than a dependency on Iceberg (avoids any potential conflicts or other issues).

Reply:
Yeah, agreed. I thought about this, too. They are unlikely to change as they are in the spec.